.github/CODEOWNERS

```diff
@@ -89,18 +89,6 @@
 # PRLabel: %OpenTelemetry
 /sdk/tracing/azotel           @jhendrixMSFT @rickwinter
 
-###########
-# Eng Sys
-###########
-/eng/                         @benbp @weshaggard
-/eng/common/                  @Azure/azure-sdk-eng
-/.github/workflows/           @Azure/azure-sdk-eng
-/eng/config.json              @richardpark-msft @jhendrixMSFT @rickwinter @chlowell @gracewilcox
-/.config/1espt/               @benbp @weshaggard
-
-# Add owners for notifications for specific pipelines
-/eng/common/pipelines/codeowners-linter.yml       @rickwinter
-
 # Add owners for go sdk tool
 /eng/tools/generator          @ArcturusZhang @lirenhe @tadelesh @raych1
 /eng/tools/internal           @ArcturusZhang @lirenhe @tadelesh @raych1
```

Comment:
Please keep these in the same area as the other eng/tools/.... entries. Makes it easier to find.

Reply:
I'll move them down to just below the Eng Sys block.